src/yunohost/certificate.py

```diff
@@ -803,6 +803,10 @@ def _enable_certificate(domain, new_cert_folder):
     for service in ("postfix", "dovecot", "metronome"):
         _run_service_command("restart", service)
 
+    if os.path.isfile('/etc/yunohost/installed'):
+        # (We don't do this yet if postinstall is not finished yet)
```

Comment:
```suggestion
        # regen nginx conf to be sure it integrates OCSP Stapling
        # (We don't do this yet if postinstall is not finished yet)
```

Integrate reason on why we do that so we don't forget in 2 years.

Reply:
done